tools/test-proxy/explore-wiremock/wiremock.md

```diff
@@ -0,0 +1,68 @@
+# Using Wiremock as the azure-sdk proxy server
```

Comment:
Is this something you would still like to keep for history purposes?

Reply:
Yeah that was my intent.